test/test_retry_download.py

```diff
@@ -1,46 +0,0 @@
-import os
```

Comment:
Did this test fail and it was too difficult to repair or is the use-case (which it is testing) no longer relevant?

Reply:
I hope you do not think of me as the kind of developer who would delete a test just because I could not work out how to fix it! :wink:

The functionality is no longer relevant - if a slice fails to download then it will be deleted, so it is not possible for the retry to resume where it finished.

Reply:
Yes, sure, I **know** that you're not that kind of developer.